pkg/sql/mutations/mutations.go

```diff
@@ -292,8 +296,12 @@ func statisticsMutator(
 				// Should not happen.
 				panic(err)
 			}
+			j, err := tree.ParseDJSON(string(b))
```

Comment:
Can this stuff go away now due to SerializeForDisplay?

Reply:
Unfortunately, no. `SerializeForDisplay` adds type annotations for literals, and statistics must be `:::JSON` not `:::STRING`